components/button/BackBtn.tsx

```diff
@@ -0,0 +1,15 @@
+import "./Button.module.css";
+import styles from "./Button.module.css";
+
+function BackBtn() {
```

Comment:
To avoid upcoming changes, I would add `onClick` as property and forward it to your `<button>`.
This will be visible in StoryBook too.